panpipes/panpipes/pipeline_ingest/pipeline.yml

```diff
@@ -206,48 +94,29 @@ custom_genes_file: resources/qc_genelist_1.0.csv
 calc_proportions: hb,mt,rp
 score_genes: MarkersNeutro
 
+#-------------------------
+# cell cycle action
+#-------------------------
+ccgenes: default
+
 #------------------------
 # Plot QC 
 #------------------------
-# all metrics should be inputted as a comma separated string e.g. a,b,c
-
-# base of the plots, it is a column in the obs, can be a comma separated string of multiple categorical 
-# plotqc_grouping_var: sample_id,rna:channel,prot:sample
-# normally is the channel also referred to "sample_id"
-# if left blank, the base of the plot will be the `sample_id` of your submission file
 plotqc_grouping_var: orig.ident
 
 
 # ------------
 # Plot RNA QC metrics
 # ------------
-# other cell covariates in rna .obs
 plotqc_rna_metrics: doublet_scores,pct_counts_mt,pct_counts_rp,pct_counts_hb,pct_counts_ig
 
-
 # ------------
 # Plot PROT QC metrics 
 # ------------
-# requires prot_path to be included in the submission file
```

Comment:
leave this sentence in:

requires prot_path to be included in the submission file	
# all metrics should be inputted as a comma separated string e.g. a,b,c

Reply:
not as a header of course 😭